### DISCUSSION TITLE
401 Unauthorized - The current user has insufficient permissions to perform the requested o (truncated...)

### DISCUSSION THREAD

**Discussion by ettorebarion** [imdhemy/laravel-in-app-purchases#339] [Q&A]
Hi Dhemy,

I'm getting this error when the request from Google arrives on my backend. Maybe I need to wait for 36 hours after creating my credentials or I missing anything? Thanks.

 {
May 16 08:14:14 vps45875 laravel[572339]:   "error": {
May 16 08:14:14 vps45875 laravel[572339]:     "code": 401,
May 16 08:14:14 vps45875 laravel[572339]:     "message": "The current user has insufficient permissions to perform the requested o (truncated...)
May 16 08:14:14 vps45875 laravel[572339]:  {"exception":"[object] (GuzzleHttp\\Exception\\ClientException(code: 401): Client error: `GET https://androidpublisher.googleapis.com/androidpublisher/v3/applications/com.XXXXXXXX.XXXXXXX/purchases/subscriptions/proprietario_30dias/tokens/gbeaphpmoalofjbblclhkmcb.AO-J1OwlPjFa7a3P8l4j4ywUnCzFxrVrGQaPz-gmCOwMP3UXbixTxcLfnJrMZH26xZMHaqfO6MuNPWG6DXgjVi-5h4M6itGx_Hi5arTd_M6dHcEwCvi7te0` resulted in a `401 Unauthorized` 

If I make a GET to this URL, I see more information:

{
    "error": {
        "code": 401,
        "message": "**Request is missing required authentication credential. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project**.",
        "errors": [
            {
                "message": "Login Required.",
                "domain": "global",
                "reason": "required",
                "location": "Authorization",
                "locationType": "header"
            }
        ],
        "status": "UNAUTHENTICATED",
        "details": [
            {
                "@type": "type.googleapis.com/google.rpc.ErrorInfo",
                "reason": "CREDENTIALS_MISSING",
                "domain": "googleapis.com",
                "metadata": {
                    "service": "androidpublisher.googleapis.com",
                    "method": "androidpublisher.SubscriptionPurchasesService.Get"
                }
            }
        ]
    }

**Comment by imdhemy** [comment]
Hi @ettorebarion 
This could help :) #94
